ephios/core/utils.py

```diff
@@ -1,9 +1,17 @@
+import logging
```

Comment:
this file is a bit weird IMO. why do we have a utils.py when the whole extra module basically provides utils?

Reply:
I think there is a difference between library-like functions, like custom widgets and mixins, and actual business logic. This file  - in `ephios.core` - has business logic, so I don't think it's "extra". 

Both artifacts in this module hook into one thing happening and trigger another thing. We could rename it to `services.py`, or, as we should probably avoid growing a random assortment of things in one file, could create an `ephios.core.services` package containing this sort of stuff. That package might also house the `notifications` package, and some of the async setup later on.